assets/sass/components/analytics-4/audience-segmentation/_googlesitekit-audience-selection-panel.scss

```
@@ -208,6 +195,26 @@
 		.googlesitekit-audience-creation-error-notice {
 			margin-bottom: 12px;
 		}
+
+		.googlesitekit-audience-selection-panel__audience-creation-notice-dismiss {
+			display: flex;
+			justify-content: flex-end;
```

Can't we simply use `text-align` here instead of creating a flex container?


```suggestion
			text-align: end;
```

---

That's a nice suggestion. Thanks, updated it 🙂